app/_src/using-mesh/managing-ingress-traffic/gateway-api.md

```diff
@@ -262,19 +271,51 @@ spec:
 and then reference it in your `Gateway`:
 
 ```yaml
-apiVersion: gateway.networking.k8s.io/v1beta1
+apiVersion: gateway.networking.k8s.io/v1
 kind: Gateway
 metadata:
   name: kuma
   namespace: default
 spec:
   gatewayClassName: kuma-cross-mesh
+  listeners:
+    - name: proxy
+      port: 8080
+      protocol: HTTP
 ```
 
-### Multi-zone
+### Multi-zone Deployments
 
+{% if_version lte:2.6.x %}
 Gateway API isn't supported with multi-zone deployments, use {{site.mesh_product_name}}'s [`MeshGateways`](/docs/{{ page.version }}/using-mesh/managing-ingress-traffic/builtin-listeners)/[`MeshHTTPRoute`](/docs/{{ page.version }}/policies/meshhttproute)/
 [`MeshTCPRoute`](/docs/{{ page.version }}/policies/meshtcproute) instead.
+{% endif_version %}
+{% if_version gte:2.7.x %}
+The Gateway API supports multi-zone deployments, but with some limitations:
+
+- Gateway API resources like `Gateway`, `ReferenceGrant`, and `HTTPRoute` must be created in non-global zones. They are specific to the zone they reside in.
+- Only services deployed within the same Kubernetes cluster, such as the `HTTPRoute`, can be referenced as the `backendRef`.
+
+{% capture backendref-limitation %}
+{% tip %}
+This is temporary limitation, which will be lifted when [work on allowing targeting {{site.mesh_product_name}}'s `MeshService`s in `backendRef`](https://github.com/kumahq/kuma/issues/9894) will be completed.
+{% endtip %}
+{% endcapture %}
+{{ backendref-limitation | indent }}
+
+Here's an example scenario that describes how you could configure multi-zone deployments with the Gateway API. In this example, you have the following resources:
+* Two zones:
+  * `zone-1` is in a Kubernetes cluster
+  * `zone-2` is in a separate Kubernetes cluster
```

Comment:
so both of them are in separate cluster right? (a bit confusing that only in `zone-2` you mention that it's in a separate cluster) so maybe:

```suggestion
* Two zones (`zone-1` and `zone-2`) in separate Kubernetes clusters
```

?